app/Helpers/Helper.php

```diff
@@ -40,3 +42,12 @@ function durationToSpan($duration): string {
 
     return $return;
 }
+
+function userTime(null|Carbon|\Carbon\Carbon|string $time=null, ?string $format=null, bool $iso=true): string {
+    $format = $format ?? __('time-format');
+    $time = $time instanceof \Carbon\Carbon ? $time : Carbon::parse($time);
+    if ($iso) {
+        return $time->tz(auth()->user()->timezone)->isoFormat($format);
+    }
+    return $time->tz(auth()->user()->timezone)->format($format);
```

Comment:
why not isoFormat everytime?

Reply:
Because isoFormat is only used in our translations (I'm not really sure why) and in my opinion, `->format()` is generally the cleaner version.